scripts/provision-all

```diff
@@ -17,6 +17,13 @@ sudo qubesctl --show-output --skip-dom0 --targets sys-firewall state.sls sd-sys-
 echo "Set up dom0 config files, including RPC policies, and create VMs"
 sudo qubesctl --show-output state.highstate
 
+echo "Setup sd-log-buster-template vm first"
+sudo qubesctl --show-output --skip-dom0 --targets sd-log-buster-template state.highstate
+# Provision whonix-gw-15 with log additions because it isn't tagged with sd-workstation (we don't want it removed after a make clean)
+sudo qubesctl --show-output --skip-dom0 --targets whonix-gw-15 state.highstate
+sudo qubesctl --show-output --skip-dom0 --targets whonix-gw-15 state.sls sd-whonix-template-files
```

Comment:
This line is not necessary: `sd-whonix-template-files` is specified in `sd-worksation.top` so will be applied as part of the qubesctl invocation of the line directly above.

Reply:
Sadly it did not for me when I tried. I had to add that line to get it working :(

Reply:
Can you try again? Based on my local testing, removing L24 works. If the issue is the rebooting of `whonix-gw-15`, it may be best to `qvm-shutdown --wait sd-whonix` here intead of re-applying a salt file that has already been applied, as it will be more reliable

Reply:
`sd-whonix` or `whonix-gw-15` ?

Reply:
woops sorry, `whonix-gw-15`, _if_ it is required